ops/pebble.py

```diff
@@ -1069,6 +1069,8 @@ def read1(self, n=-1):
 class Client:
     """Pebble API client."""
 
+    _chunk_size = 8192
```

Comment:
Why is one of the chunk sizes 8KB and the other 100KB? (block_size)

Reply:
If I'm remembering this right (couple of MRs here, this applies to at least one of them) - I think I totally forgot about the 100KB chunk size (which is what I added during initial implementation), and the 8192 was a chunk size I saw already in use elsewhere in the code that I tried to centralize for reuse later.

I'll review and likely drop the 100KB one.